maple/types.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+
+
+@dataclass(frozen=True)
+class UnknownValue:
+    """A catch-all error type when we fail to parse a value."""
+
+    value: str
+
+
+class Status(Enum):
+    """The status of a bill."""
+
+    introduced = "introduced"
+    first_committee = "first committee"
+    second_committee = "second committee"
+    chamber_1 = "chamber 1"
```

Comment:
nit: Any reason that "chamber 1" has a space and "chamber_2" has an underscore?

Reply:
Oversight.